includes/class-activitypub.php

```diff
@@ -707,16 +707,54 @@ public static function updated_postmeta( $meta_id, $object_id, $meta_key, $meta_
 	 * Register user meta.
 	 */
 	public static function register_user_meta() {
+		$blog_prefix = $GLOBALS['wpdb']->get_blog_prefix();
+
 		\register_meta(
 			'user',
-			'activitypub_also_known_as',
+			$blog_prefix . 'activitypub_also_known_as',
 			array(
 				'type'              => 'array',
 				'description'       => 'An array of URLs that the user is known by.',
-				'single'            => false,
+				'single'            => true,
 				'default'           => array(),
 				'sanitize_callback' => array( Sanitize::class, 'url_list' ),
 			)
 		);
+
+		\register_meta(
+			'user',
+			$blog_prefix . 'activitypub_description',
+			array(
+				'type'              => 'string',
+				'description'       => 'The user&#8217;s description.',
+				'single'            => true,
+				'default'           => '',
+				'sanitize_callback' => 'wp_kses_post',
```

Comment:
Core is stricter for the regular user description, [limiting it to `$allowedtags`](https://github.com/WordPress/wordpress-develop/blob/6.7.2/src/wp-includes/kses.php#L898-L902). Should we do the same?